docs/backend/schemas.md

```diff
@@ -291,13 +219,95 @@ schema = getUtility(IDexterityFTI, name=PORTAL_TYPE_NAME).lookupSchema()
 fields added to it at this stage, only the fields directly defined in your
 schema.
 
-## Field order
+
+### More info
+
+- [zope.schema](https://pypi.python.org/pypi/zope.schema) on PyPi
+- [zope.schema source code](http://github.com/zopefoundation/zope.schema) - definite source for field types and usage.
+- [zope.schema documentation](https://zopeschema.readthedocs.io)
+- [plone.schema](https://github.com/plone/plone.schema)
+
+`zope.schema` and `plone.schema` provide a very comprehensive set of {ref}`backend-fields-label` out of the box.
+Finding good documentation for them, however, can be challenging.  Here are
+some starting points:
+
+- {ref}`reference list of fields used in Plone <backend-fields-label>`
+
+
+## Advanced
```

Comment:
This is at Zope level and can confuse a little the readers (no content type but a python class). Maybe just add what this paragraph is about. Some parts in Advanced can be shared with Forms, maybe it is useful to link Forms here.

Reply:
I was thinking something like:
```
Advanced

This section uses the Zope Component Architecture to manipulate and set various aspect of the schema.

We can use a schema class to store data based on ...
```